.editorconfig

```diff
@@ -5,66 +5,45 @@ root = true
 [*]
 charset = utf-8
 end_of_line = lf
-insert_final_newline = true
-trim_trailing_whitespace = true
-
-# HTML files
-[*.html]
-indent_style = space
-indent_size = 4
-
-# CSS files
-[*.css]
-indent_style = space
-indent_size = 4
-
-
-# LESS files
-[*.less]
 indent_style = space
 indent_size = 4
-
-# SCSS files
-[*.scss]
-indent_style = tab
-indent_size = 2
+insert_final_newline = true
+trim_trailing_whitespace = true
 
 # JS files
 [*.js]
-indent_style = space
-indent_size = 4
+indent_size = 2
 
-# PHP files
-[*.php]
-indent_style = space
-indent_size = 4
+# JSON files
+[*.json]
+indent_style = tab
 
-# MD files
-[*.md]
-indent_style = space
-indent_size = 4
+# package.json
+[package.json]
+indent_size = 2
 
 # ReST files
 [*.rst]
-indent_style = space
 indent_size = 3
+max_line_length = 80
 
-# TypoScript files
-[*.typoscript]
-indent_style = space
-indent_size = 4
+# SQL files
+[*.sql]
+indent_style = tab
+indent_size = 2
 
-# YML files
-[{*.yml,*.yaml}]
-indent_style = space
+# TypoScript file
```

Comment:
I think we should keep 'files' like in every line, instead of 'file' :)

Reply:
Good catch! Fixed. (I'll still need to fix the build failures before this PR is ready for review, though.)